src/app/systelab-components/combobox/README.md

```diff
@@ -177,6 +187,7 @@ Once you have your component, you can use it in your templates.
 | expandToParentContainerHeight | boolean | false ||
 | allowEditInput | boolean | false ||
 | emptyElement | boolean | false ||
+| withFavourites | boolean | false ||
```

Comment:
Add description of the Input
Add preferenceName in the MD

Reply:
Done